pan3d/dataset_builder.py

```diff
@@ -503,12 +505,17 @@ def plot_mesh(self):
             **args,
         )
         self.plotter.view_isometric()
-        self.ctrl.reset_camera()
+        if self._cloud:
+            self.ctrl.push_camera()
+        else:
+            self.ctrl.reset_camera()
         self.ctrl.view_update()
```

Comment:
I don't think you need to do a if. You can make the same call for both. You should validate that assumption, but I would rather have a unified code base than a "complex" if for managing different strategies.

Reply:
Ok, gotcha. I confirmed that it functions properly with the default rendering mode and made the change in [8f963ff](https://github.com/Kitware/pan3d/pull/48/commits/8f963ff9c11a68d51ffbf87a287ecb88eed754a6).